packages/dashboard/src/utils/useStoryView.js

```diff
@@ -39,6 +40,7 @@ export default function useStoryView({
   isLoading = false,
   totalPages,
 }) {
+  const getAuthors = useApi((v) => v.actions.usersApi.getAuthors);
```

Comment:
Maybe I'm just feeling particularly nit-y today but `v` isn't super descriptive. maybe we just destructure `actions` off of the context?
```suggestion
  const getAuthors = useApi(({ actions }) => actions.usersApi.getAuthors);
```

Reply:
yea idk why I felt differently when I was writing this, but at the time I was thinking we shouldn't do: `state.state` or `state.actions` cus that root `state` is the `value` of the provider, not the `state`. so I was thinking it's better to use `value.actions` or `v.actions`.

Anyway, I'll just destructure cus it eliminates this convo entirely 😅